rules/opentitan_test.bzl

```diff
@@ -313,20 +313,33 @@ def opentitan_functest(
             # Copy `cw310` for each `target`. This is not a deep copy, thus we
             # also copy tags below.
             cw310_ = cw310_params() if not cw310 else dict(cw310.items())
-            if "test_rom" in target:
+            cw310_["tags"] = [t for t in cw310_["tags"]]
```

Comment:
Nit: It's difficult to tell whether we're mutating long-lived references or owned copies. I wonder whether it would be clear and not-too-inefficient to completely avoid mutable operations on the dicts and and lists. WDYT?

For instance:
```python
some_list.append(foo)  # Mutation
some_list += [foo]     # Redefinition

some_dict[key] = value      # Mutation
some_dict |= {key: value}   # Redefinition
```

Reply:
Good point, we should keep this in mind when we start cleaning up this file.